.github/workflows/golangci-lint.yml

```diff
@@ -18,7 +18,7 @@ jobs:
     steps:
       - uses: actions/setup-go@v3
         with:
-          go-version: 1.17
+          go-version: 1.19
```

Comment:
1.21 isn't supported by golangci-lint ?

Reply:
Updated golangci-lint version to 1.21. This should work